internal/api/v1/user/configs/configs.go

```diff
@@ -0,0 +1,43 @@
+package configs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/go-resty/resty/v2"
+)
+
+const route = "api/v1/configs"
```

Comment:
just a thought but don't we want a singular here? `api/v1/config`?

Reply:
![image](https://github.com/user-attachments/assets/2a354665-313e-4495-8d7d-4952d24cab8e)